README.md

```diff
@@ -55,22 +52,26 @@ Clone the Sitecore.HabitatHome.Commerce repository locally - default settings as
   
 ### 2 Deploy Solution
 Note: If you have not yet done so, deploy the base HabitatHome.Content solution. See README.md in Sitecore.HabitatHome.Content
-You can run the `quick-deploy` gulp task from the root of the HabitatHome.Content solution for a quicker deploy that excludes post deploy actions or Unicorn synchronization.
+You can run the `Quick-Deploy` gulp task from the root of the HabitatHome.Content solution for a quicker deploy that excludes post deploy actions or Unicorn synchronization.
 
-To deploy the HabitatHome.Commerce solution, from the root of the solution
+To deploy the **HabitatHome.Commerce** solution, from the root of the solution
 
-`npm install`
-`node_modules\.bin\gulp initial`
-> gulp **initial** only needs to be executed successfully during the initial deployment. Subsequent deployments can be made by running the default gulp task (gulp with no parameters). 
+`.\build.ps1 -Target Initial`
+
+_Notes:_
+
+- If the error occurs while Unicorn publishing to web, just simply re-sync by the following command
+    `.\build.ps1 -Target Initial-From-Sync-Unicorn`
+- Build target **Initial** only needs to be executed successfully during the initial deployment. Subsequent deployments can be made by running the **Default** Cake build target: `.\build.ps1` (without target specification).
 
 ### 3 Deploy Engine
 
 The next step will deploy Habitat Home's custom Commerce Engine with its relevant plugin and load the catalog, inventory and promotions.
 
 _Notes:_
-* If you want to use your own engine suffix rather than `habitat`, you need to update it in `deploy-commerce-engine.ps1`
-* If you want to use your own databases rather than `habitat_Global`, you need to update it in `\src\Project\HabitatHome\engine\wwwroot\bootstrap\Global.json`
-* If you want to use your own databases rather than `habitat_SharedEnvironments`, you need to update it in `\src\Project\HabitatHome\engine\wwwroot\data\Environments\Plugin.SQL.PolicySet-1.0.0.json`
+* If you want to use your own engine suffix rather than `habitathome`, you need to update it in `deploy-commerce-engine.ps1`
+* If you want to use your own databases rather than `habitathome_Global`, you need to update it in `\src\Project\HabitatHome\engine\wwwroot\bootstrap\Global.json`
```

Comment:
At lines 37-42 of `deploy-commerce.engine.ps1` I modify the Global.json file to change the host and admin user/password. It wouldn't be too difficult I imagine to change replace the database name here with the selected prefix?